python/requirements-wheel-build.txt

```diff
@@ -1,6 +1,7 @@
 cython>=0.29.11
 numpy==1.16.6; python_version < "3.9" and platform_machine != "aarch64"
-numpy==1.19.4; python_version >= "3.9" or platform_machine == "aarch64"
-pandas<1.1.0; python_version < "3.8"
```

Comment:
Is this deletion intentional?

Reply:
Yes, we shouldn't require pandas at build time.